plio/models.py

```diff
@@ -6,6 +6,20 @@
 from plio.config import plio_status_choices, item_type_choices, question_type_choices
 
 
+class Image(SafeDeleteModel):
+    _safedelte_policy = SOFT_DELETE
+
+    image_url = models.ImageField("Image", upload_to="images")
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = "image"
+
+    def __str__(self):
+        return f"Image id {self.id}"
```

Comment:
consistency with others
```suggestion
        return f"{self.id}: {self.url}"
```